VTParameters:
  metricFile: "../log/metrics"
  trajFile: "../log/trajectory"
  mapFile: "../mesh/map.ply"
  overallMapVoxelSize: 0.5
  exploredAreaVoxelSize: 0.3
  exploredVolumeVoxelSize: 0.5
  transInterval: 0.2
  yawInterval: 10.0
  overallMapDisplayInterval: 2
  exploredAreaDisplayInterval: 1
  OdoTopic: "/ground_truth/state"
  LaserTopic: "/gbplanner_node/surface_pointcloud"
  TimeTopic: "/runtime"
